### PR TITLE
fix page sections layout

### PR DESCRIPTION
**Pull request by victordit**

### CODE REVIEW

File: src/Resources/views/Shop/Page/show.html.twig [hunk 1]
```diff
@@ -20,14 +20,11 @@
...
         <div class="ui dividing header"></div>
-        <strong>
-            {{ 'bitbag.ui.sections'|trans }}:
-        </strong>
         <div class="bitbag-page-sections">
             {% include '@BitBagCmsPlugin/Shop/Page/Show/_sections.html.twig' %}
         </div>
     </div>
-    {% if null != page.products %}
+    {% if (page.products|length) > 0 %}
```
Comment (bitbager) [inline src/Resources/views/Shop/Page/show.html.twig]: Can you remove the bracket?

Comment (victordit) [reply src/Resources/views/Shop/Page/show.html.twig]: yes, but why ? fix CS?

Comment (bitbager) [reply src/Resources/views/Shop/Page/show.html.twig]: Because of our coding standard, we don't use brackets if not necessary.

Comment (victordit) [reply src/Resources/views/Shop/Page/show.html.twig]: ok, done! :)

**Comment by bitbager** [comment]
Thanks!

**Pull request merged**